ovos_phal_plugin_hotkeys/__init__.py

```diff
@@ -11,20 +11,89 @@
 # limitations under the License.
 #
 import json
+from json.decoder import JSONDecodeError
+from os.path import exists
+
 from ovos_bus_client.message import Message
 from ovos_plugin_manager.phal import PHALPlugin
 from ovos_utils.log import LOG
 
 import ovos_phal_plugin_hotkeys.keyboard as keyboard
+from ovos_phal_plugin_hotkeys.boards import WM8960, SJ201
+
+# Add boards from ovos-i2csound here, or custom boards from boards.py
+# Be sure to add the import if getting from boards.py
+KNOWN_BOARDS = ["wm8960", "SJ201"]
 
 
 class HotKeysPlugin(PHALPlugin):
     """Keyboard hotkeys, define key combo to trigger listening"""
 
     def __init__(self, bus=None, config=None):
         super().__init__(bus=bus, name="ovos-PHAL-plugin-hotkeys", config=config)
+        # Assign a value to 'autoconfig_file'
+        # Can either contain a single line with values from 'ovos-i2csound',
+        # Or can be valid json with 'key_up' and or 'key_down'
+        self.autoconfig_file = self.config.get(
+            "autoconfig_file", "/home/ovos/.config/mycroft/i2c_platform")
```

Comment:
can this be moved to the autoconfig method directly, instead of being an argument to it?

avoids the need to save the variable